actions/workflows/st2workroom_st2enterprise_test.yaml

```diff
@@ -56,9 +56,24 @@
           DISABLE_HUBOT: "true"
           CI: "true"
         hosts: "{{hostname}}"
-        cmd: "curl -sSL https://stackstorm.com/install.sh -o /tmp/bootstrap.sh && chmod +x /tmp/bootstrap.sh && /tmp/bootstrap.sh -a /tmp/answers.yaml"
```

Comment:
So previously it didn't fail?

I'm a bit confused since we used && and not ; so it should have failed...